.github/workflows/dco.yaml

```
@@ -0,0 +1,23 @@
+name: DCO
```

[IMO] How about specifying this DCO is a workaround?

```suggestion
name: DCO-workaround
```